modules/pipeline/providers/actionmgr/provider.go

```diff
@@ -62,6 +66,15 @@ func (s *provider) Init(ctx servicehub.Context) error {
 	s.defaultActionsCache = make(map[string]apistructs.ExtensionVersion)
 	s.pools = goroutinepool.New(s.Cfg.PoolSize)
 	s.bdl = bundle.New(bundle.WithAllAvailableClients())
+	go func() {
+		if s.EdgeRegister.IsEdge() {
+			return
+		}
+		err := s.actionService.InitAction(FilePath)
+		if err != nil {
+			s.Log.Errorf("failed to InitExtension error %v", err)
```

Comment:
what will happen if InitExtension failed? does one pipeline can start&run correctly at edge side?

Reply:
Synchronization is from the center to the edge. Only the center will start the InitAction, and then call the save method to synchronize to the edge

Reply:
I changed the InitAction method. Only a single action synchronization will fail, and the overall InitAction method will not fail